redwood/plugins.yml

```
@@ -0,0 +1,30 @@
+# This is the index for all Tutor plugins used by edunext.
+# It is available at the following location: https://raw.githubusercontent.com/eduNEXT/tutor-plugin-indexes/main/redwood/plugins.yml
+# The default plugins from the overhang index: https://overhang.io/tutor/main/redwood/plugins.yml
+
+- name: codejail
+  src: 'git+https://github.com/eduNEXT/tutor-contrib-codejail.git@v18.0.0'
+  url: 'https://github.com/eduNEXT/tutor-contrib-codejail.git'
+  author: edunext <technical@edunext.co>
+  maintainer: edunext <technical@edunext.co>
+  description: >
+    Tutor plugin that enables execution of untrusted code in secure sandboxes
+    using an external service based on the codejail library.
+
+- name: mfe_extensions
+  src: 'git+https://github.com/eduNEXT/tutor-contrib-mfe-extensions.git@vredwood'
```

the branch is named `redwood` so it should be with out the `v` rigth?
```suggestion
  src: 'git+https://github.com/eduNEXT/tutor-contrib-mfe-extensions.git@redwood'
```